components/push/src/subscriber.rs

```diff
@@ -11,6 +11,8 @@ use crate::config::PushConfiguration;
 use crate::crypto::{Crypto, Cryptography, KeyV1 as Key};
 use crate::storage::{Storage, Store};
 
+use crate::msg_types::SubscriptionChanged;
```

Comment:
I'm not a big fan of using Protobuf data classes in business code, but we already do that in some other crates so 🤷‍♂

Reply:
That's fair. Moving this to `libs.rs` would mean moving some of the channel looping stuff to convert it to a `SubscriptionChanged` in there. I'll try to nail this in a follow-up.

Reply:
Well, as long as we're discussing bike shed colors, I'd be ok with naming it `PushSubscriptionChange` since it ties both to the DOM message and notes what's happening. Might make finding and associating function a bit easier, but that's just me.

Reply:
Oh no, I merged the PR before seeing this! I'll create a follow-up soon. I agree that following the naming conventions already created make this easier to grok.